RZAssemblage/RZArrayAssemblage.h

```diff
@@ -0,0 +1,25 @@
+//
+//  RZArrayAssemblage.h
+//  RZAssemblage
+//
+//  Created by Brian King on 3/21/15.
+//  Copyright (c) 2015 Raizlabs. All rights reserved.
+//
+
+#import "RZAssemblage.h"
+
+@interface RZArrayAssemblage : RZAssemblage
+
+- (instancetype)initWithArray:(NSArray *)array;
+
+- (instancetype)initWithArray:(NSArray *)array representingObject:(id)representingObject;
+
+@property (strong, nonatomic) id representedObject;
```

Comment:
Is this `readwrite` simply because `RZPropertyAssemblage` sets it, or is it truly valuable to be able to set this?